ipn/ipnlocal/local.go

```diff
@@ -5830,7 +5830,14 @@ func (b *LocalBackend) setNetMapLocked(nm *netmap.NetworkMap) {
 	b.pauseOrResumeControlClientLocked()
 
 	if nm != nil {
-		b.health.SetControlHealth(nm.DisplayMessages)
+		messages := make(map[tailcfg.DisplayMessageID]tailcfg.DisplayMessage)
```

Comment:
We know the number of elements upfront, would it make sense to pass the capacity hint?

Reply:
I prefer to omit that when it's not necessary for performance, because a) it adds noise to the code and b) getting it wrong can be worse than not doing it at all. In this case, the size is likely to be 0 or 1, so I don't think it's necessary? Very happy to add it if you think otherwise though!

Reply:
I personally don't find it more noisy, and appreciate the habit of always doing it when it's straightforward (like here) instead of evaluating necessity every time. But I agree it's not super important here, since in most cases it will be zero - so up to you